components/brave_wallet_ui/components/shared/style.tsx

```diff
@@ -56,6 +56,11 @@ export const AssetIconFactory = styled.img.attrs<AssetIconProps>(props => ({
   loading: 'lazy'
 }))
 
+export const AssetIconIframe = styled.iframe.attrs<AssetIconProps>(props => ({
+  src: `chrome-untrusted://nft-display?imageUrl=${encodeURIComponent(props.icon || '')}`,
```

Comment:
Is this one still working?

Reply:
Yes, it is still working.

Reply:
I'm a bit confused about how this thing is working if we are using a postMessage API. How the `imageUrl` is being passed to the untrusted component in this case?